packages/types/src/index.ts

```diff
@@ -122,6 +122,21 @@ export type ChangelogFunctions = {
   getDependencyReleaseLine: GetDependencyReleaseLine;
 };
 
+export type GetAddMessage = (
+  changeset: Changeset,
+  commitOptions: null | Record<string, any>
```

Comment:
it would be more true if this would get annotated as `any`
```suggestion
  commitOptions: any
```